src/components/Circular Support Indicator/CircularSupportIndicator.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import CircularSupportIndicator from './CircularSupportIndicator';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Components/Circular Support Indicator',
+  component: CircularSupportIndicator,
+} as ComponentMeta<typeof CircularSupportIndicator>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof CircularSupportIndicator> = (args) => (
+  <CircularSupportIndicator {...args} />
+);
+
+export const CircularIndicator = Template.bind({});
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
+CircularIndicator.args = {
+  score: 1,
+  labels: ['disputing', 'balanced', 'supporting'],
+  tooltips: [],
```

Comment:
Maybe we could add additional type like WithTooltip, to display that there is more options for this component, that it can be displayed with tooltips. Maybe you could as well add one more like BiggerScore, to display how it looks with bigger value than 1.

Reply:
There's a validation in SupportIndicator component that would prevent bigger score than 1. I mean any number that comes from back end and is bigger than 1 would display not more than 100%. I also thought about throwing an error for such a situation.

Reply:
It's written as `if (score < 0.4) ...` and `if (score >= 0.6)`, so in case the backend sends a score of 50 for some reason it will get interpreted as 1, I don't think we need an error for that.